modules/serverless-logic/pages/cloud/operator/install-serverless-operator.adoc

```diff
@@ -29,6 +31,8 @@ To remove the operator on OpenShift refer to the "link:{openshift_operator_unins
 
 == {product_name} Operator Kubernetes installation
 
+=== Install
+
 To install the operator on Kubernetes refer to the  "link:{kubernetes_operator_install_url}[How to install an Operator from OperatorHub.io]" from the OperatorHub's documentation.
```

Comment:
I think this installation procedure from OperatorHub.io we must remove, because it's not updated, and also don't point to the OSL 1.33.0.

![image](https://github.com/user-attachments/assets/122444c4-be55-4149-9a0b-0b6a4e0dda1c)


I think we should keep only

OpenShift Serverless Logic Operator OpenShift installation
OpenShift Serverless Logic Operator Manual Installation